### ISSUE TITLE
[BUG] certain python3 lib will not execute on internal: run shell path (local) 

### ISSUE THREAD

**Issue by MRmP** [bitfocus/companion#1210]
Trying to run a simple python3 script with the internal: run shell path (local) module
Currently running 2.1.1 (e71f785-2526) raspberry pi companion build from the image available

If script is referencing the pythonosc library (https://pypi.org/project/python-osc/) with the line 
`from pythonosc import udp_client`
companion rejects with the error: 
`Shell command failed. Guru meditation: {"killed":false,"code":1,"signal":null,"cmd":"/home/pi/test.py"}`

I've tried to run the py script from another bash script, and let companion just trigger the bash script without any success, same error

**Comment by matthewpurcell** [comment]
This is a bit of a late reply, but I had the exact same issue today and spent a good few hours resolving it. Thought it would be worthwhile posting here in case others encounter the same issue.

The issue seems related to Python not finding the module (I determined this by putting `>> /Users/matthew/tmp/test.log 2>&1` onto the end of the command in Companion so it would output the error to a file, since the Companion logs don't provide any error messages when an execution fails. Indeed, Python was complaining that it could not find the module...but the script works just fine if I run it directly using `python3` in Terminal.

My next thought was different Python versions being installed on the system. I have Python 3.7, 3.8, and 3.9 installed - but use Python 3.8 (for module compatibility reasons). Even though that is set as my default/current version (when using the `python3` command in Terminal. As such, I `pip install`ed the module in Python 3.8. So I changed the shebang line in the script to specifically point to the `python3` binary for `3.8`:

`#!/usr/bin/env /Library/Frameworks/Python.framework/Versions/3.8/bin/python3`

...and it started working with no errors.

Hope this helps others!
